driver/normalizer/normalizer.go

```diff
@@ -0,0 +1,266 @@
+				  ],
+				  body=[
+					Expr(value=Name(id='t', ctx=Load())
+					) ] )
+				   ]
+			  )"
+	PR #59: Subscript ->
+			a[0] -> Index value=NumLiteral
+			a[1:2] -> Slice lower=NumLiteral upper=NumLiteral
+			a[1:2,3:4] -> ExtSlice -> [Slice, Slice]
+
+	PR #62 & 63:	- FunctionDefArg (a)
+				- FunctionDefArgDefaultValue (a = 3)
+				- FunctionDefArgAnnotation (a: int)
+				- FunctionDefVarArgsList (*args)
+				- FunctionDefVarArgsMap (**kwargs)
+	PR #63: Lambda
+
+	// Noop (decently parsed by the rules even if unorthodox):
+
+	"else" clauses for for/while/try -> Added as "IfElse" child nodes of the for/while/try
+	"stride" third element in slices
+
+ */
+
+ // TODO: all the "orelse" subnodes of for/while/try as "IfElse" childs
+ // TODO: add the "stride", third element of slices in some way once we've the index/slice roles
+var AnnotationRules = On(Any).Self(
+	On(Not(HasInternalType(pyast.Module))).Error("root must be Module"),
+	On(HasInternalType(pyast.Module)).Roles(File).Descendants(
+		// Comparison operators
+		// TODO: only apply when children of any test node types?
+		On(HasInternalType(pyast.Eq)).Roles(OpEqual),
+		On(HasInternalType(pyast.NotEq)).Roles(OpNotEqual),
+		On(HasInternalType(pyast.Lt)).Roles(OpLessThan),
+		On(HasInternalType(pyast.LtE)).Roles(OpLessThanEqual),
+		On(HasInternalType(pyast.Gt)).Roles(OpGreaterThan),
+		On(HasInternalType(pyast.GtE)).Roles(OpGreaterThanEqual),
+		On(HasInternalType(pyast.Is)).Roles(OpSame),
+		On(HasInternalType(pyast.IsNot)).Roles(OpNotSame),
+		On(HasInternalType(pyast.In)).Roles(OpContains),
+		On(HasInternalType(pyast.NotIn)).Roles(OpNotContains),
+
+		// Aritmetic operators
+		On(HasInternalType(pyast.Add)).Roles(OpAdd),
+		On(HasInternalType(pyast.Sub)).Roles(OpSubstract),
+		On(HasInternalType(pyast.Mult)).Roles(OpMultiply),
+		On(HasInternalType(pyast.Div)).Roles(OpDivide),
+		On(HasInternalType(pyast.Mod)).Roles(OpMod),
+		// TODO: currently without mapping in the UAST
+		//On(HasInternalType(pyast.FloorDiv)).Roles(OpDivide),
+		//On(HasInternalType(pyast.Pow)).Roles(???),
+		//On(HasInternalType(pyast.MatMult)).Roles(???),
+
+		// Bitwise operators
+		On(HasInternalType(pyast.LShift)).Roles(OpBitwiseLeftShift),
+		On(HasInternalType(pyast.RShift)).Roles(OpBitwiseRightShift),
+		On(HasInternalType(pyast.BitOr)).Roles(OpBitwiseOr),
+		On(HasInternalType(pyast.BitXor)).Roles(OpBitwiseXor),
+		On(HasInternalType(pyast.BitAnd)).Roles(OpBitwiseAnd),
+
+		// Boolean operators
+		On(HasInternalType(pyast.And)).Roles(OpBooleanAnd),
+		On(HasInternalType(pyast.Or)).Roles(OpBooleanOr),
+		On(HasInternalType(pyast.Not)).Roles(OpBooleanNot),
+
+		// Unary operators
+		On(HasInternalType(pyast.Invert)).Roles(OpBitwiseComplement),
+		On(HasInternalType(pyast.UAdd)).Roles(OpPositive),
+		On(HasInternalType(pyast.USub)).Roles(OpNegative),
+
+		// FIXME: boolliteral should probably be added to the UAST
```

Comment:
@smola, removed in the last commit, thanks! I've removed the integration test generated files fom the repository in this commit until the ordering works because it was cruttering a lot the history with the "changed" generated files (that weren't really changed). 

Maybe of your interest is that I also added in this commit a (WIP) temporal document (tests/todo_rules.md) with the current state of the UAST conversion based on a manual review of the UAST vs the AST and the source. I'll remove it once finished and convert it to issues / fixes / PRs in the Python driver and the SDK, but if you don't want that file there for any reason I can remove it.